recipes/pytest-pylint/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pytest-pylint" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "41d6f223ae5e0a0fbb0056e826ecdb8046be2a828eba55c0d4f66cbfd7d27168" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
```

Comment:
Looking at `setup.py`, to run it only the two above packages are necessary. The rest can be removed from the build requirements.

Reply:
I will test.

Reply:
install_requires=['pytest>=2.7', 'pylint>=1.4.5', 'six'],
Are the requirements (setup.py line 22).

Reply:
You should put the version dependencies here too, because they probably matter. `pytest >=2.7`, `pylint >=1.4.5`.

Reply:
Thank, I changed it.